packages/gestalt/src/Heading.js

```diff
@@ -85,6 +85,19 @@ export default function Heading({
   overflow = 'breakWord',
   size = '600',
 }: Props): Node {
+  const getWordBreakStyle = (): string | void => {
```

Comment:
could we reuse this between Text and HEading? helper function?

Reply:
Agree, I think for now, it's okay to dupe, since it also imports from the CSS class, sometimes importing CSS outside of a React context doesn't always work as well

Reply:
They may also diverge at some point